vpr/src/base/read_options.cpp

```diff
@@ -3088,6 +3088,11 @@ argparse::ArgumentParser create_arg_parser(const std::string& prog_name, t_optio
         .help("Writes implemented design final timing summary to the specified JSON, XML or TXT file.")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    analysis_grp.add_argument<bool, ParseOnOff>(args.generate_net_timing_report, "--generate_net_timing_report")
+        .help("Generates a net timing report for each net in the design.")
```

Comment:
Explain a little about what that is ... the delay and slack of every routed connection in a design.